packages/client-core/i18n/en/user.json

```
@@ -376,7 +379,13 @@
     "thumbnail": "Thumbnail",
     "enterAvatarUrl": "Enter Avatar URL",
     "enterThumbnailUrl": "Enter Thumbnail URL",
-    "avatarName": "Avatar Name"
+    "avatarName": "Avatar Name",
+    "done": "Done",
+    "finishedEditing": "Finished Editing",
```

```suggestion
    "finishEditing": "Finish Editing",
```

verb indicating action

---

Done!